lib/sentry/client_error.ex

```diff
@@ -34,15 +41,24 @@ defmodule Sentry.ClientError do
 
   @doc false
   @spec new(reason()) :: t
+  def new({:server_error, http_response}) do
+    %__MODULE__{reason: :server_error, http_response: http_response}
+  end
+
   def new(reason) do
-    %__MODULE__{reason: reason}
+    %__MODULE__{reason: reason, http_response: nil}
```

Comment:
It's `nil` by default, you can leave this alone:

```suggestion
    %__MODULE__{reason: reason}
```